api/scripts/wazuh-apid.py

```diff
@@ -72,8 +72,11 @@ def main(cors, port, host, ssl_context, main_logger):
         os.chmod('{0}/logs/api.log'.format(common.ossec_path), 0o660)
 
     if configuration['https']['enabled']:
-        ssl_context = ssl.SSLContext()
-        ssl_context.load_cert_chain(certfile=configuration['https']['cert'], keyfile=configuration['https']['key'])
+        ssl_context = ssl.SSLContext(ssl.PROTOCOL_TLS)
```

Comment:
The default value of the first and unique parameter in `ssl.SSLContext` is already `ssl.PROTOCOL_TLS`: https://docs.python.org/3/library/ssl.html#ssl.SSLContext

Reply:
done!